O365/connection.py

```diff
@@ -978,13 +970,14 @@ def _check_delay(self) -> None:
         self._previous_request_at = time.time()
 
     def _internal_request(
-        self, session_obj: Session, url: str, method: str, **kwargs
+        self, session_obj: Session, url: str, method: str, ignore401: bool, **kwargs
```

Comment:
Can you please define it a little bit different?

```python
def _internal_request(
        self, session_obj: Session, url: str, method: str, *, ignore401: bool = False, **kwargs
    ) -> Response:
```
This is just so if someone passes args to internal requests `ignore401` param don't receive an unintended param. Also we define a default value for it.

Reply:
You didn't add the star before the ignore401, but anyway I've merged it